src/main/java/core/basesyntax/operation/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
+        int fruitsInStorage = Storage.storage.get(fruitTransaction.getFruit());
+        int updateQuantity = fruitsInStorage + fruitTransaction.getQuantity();
+        Storage.storage.put(fruitTransaction.getFruit(), updateQuantity);
+    }
```

Comment:
According to the checklist, there should not be an empty line at the beginning of the class implementation. Please remove the empty line after the class declaration.